packages/core/src/WebviewDatadog/__utils__/getInjectedJavaScriptBeforeContentLoaded.ts

```diff
@@ -0,0 +1,19 @@
+import { formatAllowedHosts } from './formatAllowedHosts';
+
+export const DATADOG_MESSAGE_PREFIX = '[DATADOG]';
+
+export const getInjectedJavaScriptBeforeContentLoaded = (
+    allowedHosts?: string[]
+) =>
+    `
+      window.DatadogEventBridge = {
+        send(msg) {
+
+          if(typeof msg === 'undefined') throw new Error('')
```

Comment:
Could you add a comment to explain why we need this check?
When we add this check here we have to maintain it and test it when making changes. If there is no special reason, we should remove it.